"""
Написать функцию которая принимает на вход число от 1 до 100.
Если число равно 13, функция поднимает исключительную ситуации ValueError
иначе возвращает введенное число, возведенное в квадрат.
Далее написать основной код программы. Пользователь вводит число.
Введенное число передаем параметром в написанную функцию и печатаем результат,
который вернула функция. Обработать возможность возникновения исключительной ситуации,
которая поднимается внутри функции.
"""


def my_func(number):
    if number == 13:
        raise ValueError("Несчастливое число")
    else:
        return number ** 2

while True:
    try:
        user_number = int(input("Введите число от 0 до 100: "))
        break
    except ValueError:
                    print("Нужно вести целое число!")

print(f"Ваше число в квадрате равняеться: {my_func(user_number)}")
